app/scripts/lib/transaction/metrics.ts

```diff
@@ -1258,6 +1259,10 @@ function addGaslessProperties(
       token.tokenAddress.toLowerCase() === selectedGasFeeToken?.toLowerCase(),
   )?.symbol;
 
```

Comment:
Issue @bschorchit raised here is that she is finding `ETH` in `gas_paid_with`. But according to logic above, it is not possible if `ETH` is not in `transactionMeta .gasFeeTokens` which is returned from simulations.

Thus I am not sure why is this issue happening.